ONSdigital/response-operations-ui
# Create user

# Motivation and Context
This allows trusted people with the administrator password to send account creation emails to new members of staff, taking this away from being a DevOps issue

# How to test?
Spin up the accompanying branch of ras-rm-docker-dev (`new-envs-for-create-account`) and start response-operations. You should be able to go through the create account process by using the default admin password `secret` and by taking the email confirmation link out of the logs
